src/stores/store/models/testutils/faker.go

```diff
@@ -1,9 +1,9 @@
 package testutils
 
 import (
+	"github.com/consensys/quorum-key-manager/src/stores/store/models"
 	"time"
 
-	"github.com/consensys/quorum-key-manager/src/stores/store/database/models"
```

Comment:
Why did you move your models one level up, as far as I see it models do not make sense out of database/, same as migrations

Reply:
reverted